web/src/settings_users.js

```diff
@@ -633,3 +643,44 @@ export function set_up_bots() {
         settings_bots.add_a_new_bot();
     });
 }
+
+export async function fetch_presence_user_setting() {
+    if (page_params.is_spectator) {
+        return;
+    }
+
+    await new Promise((resolve) => {
+        channel.post({
+            url: "/json/users/me/presence",
+            data: {
+                status: compute_active_status(),
+                ping_only: false,
+                last_update_id: -1,
+                history_limit_days: 3650,
+            },
+            success(response) {
+                try {
+                    const data = post_presence_response_schema.parse(response);
+
+                    if (data.presences) {
+                        const updatedPresenceInfo = {};
```

Comment:
You're mixing camel case naming convention with our usual snake case. We want to be consistent, so this should be `updated_presence_info`.

Reply:
ok, I will make sure not to do this case mistake in future.
Thanks!